<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PendingRequests</class>
 <widget class="QDialog" name="PendingRequests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1198</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1198</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(white.png);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="leftMenu" native="true">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>678</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(&quot;22.png&quot;);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_24">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_19">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_25">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QFrame" name="frame_20">
           <property name="styleSheet">
            <string notr="true">border-image:url(&quot;C:/Users/Hema/Desktop/Project/22.png&quot;);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>40</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">COLOR:rgb(255, 255, 255);
font: 700 40pt &quot;Tahoma&quot;;</string>
              </property>
              <property name="text">
               <string>UMS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_39">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(close.png);
border-radius:17px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_40">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_40::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_41">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_41::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string>Add Student</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_43">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_43::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string>Students Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_42">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_42::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string>Add Course</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_47">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_47::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string>Courses Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_46">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(white.png);
padding:18px;
font: 900 11pt &quot;Arial Black&quot;;
text-align:left;
color:rgb(103,58,183);</string>
           </property>
           <property name="text">
            <string>Pending Requests</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_44">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{padding:18px;
text-align:left;
font: 900 11pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255);
}

#pushButton_44::hover{
border-image: url(white.png);
color:rgb(103,58,183);
}</string>
           </property>
           <property name="text">
            <string> Log Out</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QFrame" name="frame_21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_26">
            <property name="spacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainBody" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_27">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="HeaderFrame_6" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_45">
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(menubutton.jpg);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>17</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">*{
	font: 700 17pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Pending Requests</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_12" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="mainFrame_6" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(white.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>740</width>
             <height>580</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>874</width>
              <height>578</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>555</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
